apps/mull-ui/src/app/app.tsx

```diff
@@ -41,6 +44,15 @@ export const App = () => {
             <EventPage event={dummyEvent} prevPage={'Review'} />
           </div>
         </Route>
+        {/* Without this outter Route component, the inner components would show on all routes that are not matched */}
+        <Route exact path={[`${ROUTES.DISCOVER}`, `${ROUTES.UPCOMING}`, `${ROUTES.MYEVENTS}`]}>
```

Comment:
The more I think about it the more I think these routes should be under `/home`, since each tab is a subpage for the home page. I'm not gonna block the PR over this, but I really think what you did initially was good.

Reply:
Sure, I will change it